apps/terminal/const.py

```diff
@@ -67,6 +67,8 @@ class TerminalType(TextChoices):
     chen = 'chen', 'Chen'
     kael = 'kael', 'Kael'
     panda = 'panda', 'Panda'
+    nec = 'nec', 'Nec'
+    facelive = 'facelive', 'Facelive'
 
     @classmethod
     def types(cls):
```

Comment:
The code does not contain any irregularities or potential issues at this time. There is no suggestion for optimization since it's a static method that returns a list of terminal type strings without any parameters. However, we can optimize the docstring slightly:

```markdown

**TERMINAL TYPE**

This enum represents terminal types.

Attributes:
    - CHEN: The Chen style
    - KAEILIVE: A Necromancer who speaks Korean with an accent (translated as 'Falken')
    - PANDA: Panda species
```
Now, let's review the code again to see if there are any other differences you would like to mention.
In summary, the current version seems correct and provides all necessary documentation. No further changes to be made here.